### DISCUSSION TITLE
BigWig Track not showing up

### DISCUSSION THREAD

**Discussion by Vishvak2000** [GMOD/jbrowse-components#2123] [Q&A] [answered]
Hello! I am configuring a genome browser for a viral protein and want to add an entropy track. The protein is 400 residues long and I have an entropy track in wig format as follows:

> track type=wiggle_0 track type=wiggle_0 name="Entropy_Subtype_C" description="Entropy_C_clade"
> fixedStep start=1 chrom=chrS step=1
> 1.00
> 1.00
> 1.00
> 0.93
> ...

My chrom.sizes file is as follows:

> chrS	400

I am able to succesfully convert it into BigWig format using the UCSC wigToBigWig tool however, when I upload it as a track to JBrowse2, nothing shows up except the y-axis even after ensuring that the xy-renderer is selected.

![image](https://user-images.githubusercontent.com/63014859/125340543-6e50d700-e307-11eb-8ea7-2be916570748.png)

Any help would be appreciated!

**Comment by garrettjstevens** [comment, marked as answer]
It looks like your chrome.sizes file uses `chrS`, but in your screenshot the reference sequence name is `pre-S1_pre-S2_S`. JBrowse is looking for data for `pre-S1_pre-S2_S` in your BigWig file and not finding any, so the track is blank.

You can either change your wig file and chrome.sizes file to use `pre-S1_pre-S2_S`, or you can configure [reference name aliasing](https://jbrowse.org/jb2/docs/config_guide#configuring-reference-name-aliasing).

**Reply by Vishvak2000** [reply]
Changing the name of the chrom referenced in the chrom.sizes file and the wig file worked! Thank you.
